outputs.md

```diff
@@ -200,28 +216,28 @@ Text file of generated bandpass solutions. The first column is the frequency cha
    * **obsdec**: float of the observation's DEC in degrees
    * **zenra**: float of the zenith RA in degrees for the instrument and julian date specified
    * **zendec**: float of the zenith DEC in degrees for the instrument and julian date specified
-   * **obsx**: float number of the positive x-axis for the observation, set by dimension/2
-   * **obsy**: float number of the positive y-axis for the observation, set by elements/2
-   * **zenx**: float number of the positive x-axis for the zenith, set by dimension/2
-   * **zeny**: float number of the positive y-axis for the zenith, set by elements/2
-   * **phasera**:         FLOAT           359.849
-   * **phasedec**:        FLOAT          -26.7836
-   * **orig_phasera**:    FLOAT           0.00000
-   * **orig_phasedec**:   FLOAT          -27.0000
+   * **obsx**: float of the positive x-axis for the observation, set by dimension/2
+   * **obsy**: float of the positive y-axis for the observation, set by elements/2
+   * **zenx**: float of the positive x-axis for the zenith, set by dimension/2
+   * **zeny**: float of the positive y-axis for the zenith, set by elements/2
```

Comment:
Not sure why an axis is being referred to by a number. Can these four (`obsx`, `obsy`, `zenx`, `zeny`) be clarified a bit? If unclear for now, maybe find a way of marking for clarification later since I'd prefer this didn't hold up the merge.

Reply:
These are extents, i.e. the max value in a certain direction. Modified. Hope that clears it up.